src/scripts/main.js

```diff
@@ -1,3 +1,92 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const titles = thead.querySelectorAll('th');
+const names = titles[0];
+const position = titles[1];
+const age = titles[2];
+const salary = titles[3];
+
+const employees = tbody.querySelectorAll('tr');
+
+function sortByName([...list]) {
```

Comment:
Using the spread operator to create a new array from 'list' is unnecessary here. The 'sort' method already sorts the array in place. You can directly use 'list' without spreading it.